chapters/research-design.tex

```diff
@@ -1,288 +1,593 @@
-``sufficiently similar'' strata groups.
-Adjustments for balance variables are never necessary in RCTs,
+When it is an RCT, a randomization process constructs the control group at random
+from the population that is eligible to receive each treatment.
+When it is observational, we present other evidence that a similar equivalence holds.
+Therefore, by construction, each unit's receipt of the treatment
+is unrelated to any of its other characteristics
+and the ordinary least squares (OLS) regression
+of outcome on treatment, without any control variables,
+is an unbiased estimate of the average treatment effect.
+Cross-sectional data is simple to handle because
+for research teams do not need track anything over time.
+A cross-section is simply a representative set of observations
+taken at a single point in time.
+If this point in time is after a treatment has been fully delivered,
+then the outcome values at that point in time
+already reflect the effect of the treatment.
+
+What needs to be carefully maintained in data for cross-sectional RCTs
+is the treatment randomization process itself,
+as well as detailed field data about differences
+in data quality and loss-to-follow-up across groups.\cite{athey2017econometrics}
+Only these details are needed to construct the appropriate estimator:
+clustering of the estimate is required at the level
+at which the treatment is assigned to observations,
+and controls are required for variables which
+were used to stratify the treatment (in the form of strata fixed effects).\sidenote{
+  \url{https://blogs.worldbank.org/impactevaluations/impactevaluations/how-randomize-using-many-baseline-variables-guest-post-thomas-barrios}}
+\textbf{Randomization inference} can be used
+to esetimate the underlying variability in the randomization process
+(more on this in the next chapter).
+\textbf{Balance checks} are often reported as evidence of an effective randomization,
+and are particularly important when the design is quasi-experimental
+(since then the randomization process cannot be simulated explicitly).
+However, controls for balance variables are usually unnecessary in RCTs,
 because it is certain that the true data-generating process
-has no correlation between the treatment and the balance factors.\sidenote{\url{https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments}}
-However, controls for imbalance that are not part of the design
-may reduce the variance of estimates, but there is debate on
-the importance of these tests and corrections.
-
+has no correlation between the treatment and the balance factors.\sidenote{
+  \url{https://blogs.worldbank.org/impactevaluations/should-we-require-balance-t-tests-baseline-observables-randomized-experiments}}
+
+Analysis is typically straightforward with a strong understanding of the randomization.
+A typical analysis will include a decription of the sampling and randomization process,
```

Comment:
I think these are the basics, no? what else would you add?

Reply:
I mean describing the sampling and randomization processes go in a section of the paper that does not yet include data analysis. This will only be present in the data analysis through descriptive tables (summary stats, balance tests, etc).

What is not clear to me is why "description of the sampling and randomization process" is considered data analysis.